### ISSUE TITLE
Test Java 11 JCT on updateability

### ISSUE THREAD

**Issue by grthor** [jcryptool/core#166]
The Java 11 JCT with JRE should be tested on updateability. 
I tested the updateability in September, but since then we have changed some stuff (especially the thirdparty Repository), so I would like to test it again, not to run into problems when we release version 1.1.

I would do it, so I assign myself. But it can only be tested when the Java 11 JCT is ready.

**Comment by simlei** [comment]
@grthor since [Weekly-Build--2020-02-29](https://github.com/jcryptool/core/releases/tag/Weekly-Build--2020-02-29), Java 11 JCT is ready :)

**With respect to thirdparty feature updateability**

This is certainly worth investigating; it is a level of transitivity more than our own plug-ins since the thirdparty artifacts are part of the target platform. Still, I think it is arguable whether this has to be part of version 1.0 if it poses too much of a problem, since thirdparty artifacts ought to have a much slower update cycle than our own plug-ins.

[These instructions](https://github.com/simlei/org.jcryptool.thirdparty/wiki/Making-an-update-to-JRE-features,-locally-hosted) describe how to update the JRE locally (without updating the central JRE binaries, using `http://localhost/...` update site). There are also scripts that perform a version bump of the JRE features across the thirdparty and core repositories (under Windows, use WSL or cygwin; observe if unsure e.g. by running it on a clean-state branch)


A "true" test of updateability of thirdparty features would require us to have multiple versions of org.jcryptool.thirdparty.jre in our github-hosted sudo-p2-repository at https://simlei.github.io/org.jcryptool.thirdparty/site/jre. We already get warnings about hosting big binaries on github within that project. I would argue that testing this with `http://localhost/`-ed jre bundles that have got their versions bumped will suffice since the transport protocol is identical.

**Comment by grthor** [comment]
tl;dr It works.

And here are some additional notes:
- Increasing only the version number of a single feature or plugin is not possible. All version numbers have to be increased. For the switch from version 1.0.0 to 1.0.1 this is described in our wiki: https://github.com/jcryptool/core/wiki/Create-an-update-for-the-update-site (i wrote this article some time ago).
- The version numbers of Eclipse, babel, the Thirdparty repo, other external packages do not affect the update process. The packages are encapsulated into features (e.g. org.jcryptool.dependencies.feature for the Eclipse RCP). Only the feature version number matters. This allows us to update and downgrade e.g. the eclipse RCP and all other external packages.

The update process is ready for release.
